capstone/client/src/components/ListPage.js

```diff
@@ -251,6 +292,8 @@ class ListPage extends Component {
         {this.state.errorMessage ? <Alert severity="error">{this.state.errorMessage}</Alert> : null}
         {this.state.successMessage ? <Alert severity="success">{this.state.successMessage}</Alert> : null}
         <h1>Preferences</h1>
+        <ButtonGroup container id="user-lists" dangerouslySetInnerHTML={this.userLists()} onClick={this.selectList}>
```

Comment:
You shouldn't use dangerouslySetInnerHTML here. You can save the number of saved lists the user has in the state and map each list to a button in the render function. This will also allow you to better style the buttons. I think it would look nicer if there was spacing between the buttons and if they were a different color. You could experiment using the Material UI buttons with the prop color="primary" or color="secondary".